app.py

```diff
@@ -288,6 +349,9 @@ def register():
             cur.execute("INSERT INTO users(name, username, password, accountType) Values(%s, %s, %s, %s)", (name, username, password, accountType))
             mysql.connection.commit()
             flash("Registered Successfully", "success")
+            msg = Message('C4DFED Account', sender='******@gmail.com', recipients=[username])
```

Comment:
Why is the sender hardcoded here? It was there at the top config right? We should just use one place where it is configured.

Reply:
Same for every other place.